routes/signin.js

```diff
@@ -0,0 +1,99 @@
+var express = require('express');
+var app = express();
+var router = express.Router();
+var mysql = require('mysql');
+var bodyParser = require('body-parser');
+var methodOverride = require('method-override');
+//for login/logout (authentication)
+var bcrypt = require('bcryptjs');
+var jwt = require('jsonwebtoken');
+//use sendgrid
+// var sgMail = require("@sendgrid/mail");
+// var keys = require("../key");
+// sgMail.setApiKey(keys.sendgrid);
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+app.use(express.static('public'));
+app.use(methodOverride('_method'));
+
+var connection = mysql.createConnection({
+  host: 'localhost',
+
+  // Your port; if not 3306
+  port: 3306,
+
+  // Your username
+  user: 'root',
+
+  // Your password
+  password: 'password',
+  database: 'crypto_db'
+});
+
+connection.connect(function(err){
+  if(!err) {
+      console.log("Database is connected ... nn");
+  } else {
+      console.log("Error connecting database ... nn");
+  }
+  });
+  require('dotenv').config()
```

Comment:
can you put this line at the top @jace1084 so it's consistent with the other route files? 
 ```require('dotenv').config()```

Reply:
Will do!